standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1821,6 +1821,10 @@ public enum ConfVars {
             "A ZooKeeper instance must be up and running when using zookeeper Hive lock manager "),
     HIVE_TXN_STATS_ENABLED("hive.txn.stats.enabled", "hive.txn.stats.enabled", true,
         "Whether Hive supports transactional stats (accurate stats for transactional tables)"),
+    MSCK_SMALLFILES_AVG_SIZE("metastore.msck.smallfiles.avgsize", "metastore.msck.smallfiles.avgsize", (long) (16 * 1000 * 1000),
```

Comment:
if we are not able to access HiveConf HIVE_MERGE_MAP_FILES_AVG_SIZE in Metastore, we can use `conf.get("hive.merge.smallfiles.avgsize", "16000000")` compared a new property

Reply:
This is part of the design of this PR. We can add this conf here, it can also be used in any future use.